docs/arcadia-users-group/20240730-arcadiathemer/20240730-arcadiathemer.Rmd

```diff
@@ -0,0 +1,143 @@
+  geom_point() +
+  theme_arcadia() +
+  scale_color_arcadia()
+```
+
+If you want to change the color palette used, you can access this with the `palette_name` argument in the `scale` function: 
+
+```{r, palette_change}
+ggplot(data=mtcars, aes(x=hp, y=mpg, color=as.factor(cyl))) +
+  geom_point() +
+  theme_arcadia() +
+  scale_color_arcadia(palette_name = "primary")
+```
+
+We have specific font specifications for whether the represented data is categorical or numerical. In the next plot where the x-axis is different categories, you can see where the font isn't specified correctly:
+```{r, fonts}
+ggplot(data=diamonds, aes(x=cut, fill=cut)) +
+  geom_bar() +
+  theme_arcadia() +
+  scale_fill_arcadia()
+```
+
+To use the correct font type, you can specify what type of data you have on each axis by using the `x_axis_type` or `y_axis_type` arguments in the `theme_arcadia()` function: 
+```{r, fonts_correct}
+ggplot(data=diamonds, aes(x=cut, fill=cut)) +
+  geom_bar() +
+  theme_arcadia(x_axis_type = "categorical") +
+  scale_fill_arcadia()
+```
+In addition to specifying which palette to use in the `scale` function, you can also reverse the colors of the scale used with `reverse=TRUE`. You can also use other `ggplot2` or `theme` specifications on top of these functions, such as moving the position of the legend or modifying the scales to remove whitespace between the axis lines and the bars:
+```{r, additional_layers}
+ggplot(data=diamonds, aes(x=cut, fill=cut)) +
+  geom_bar() +
+  theme_arcadia(x_axis_type = "categorical") +
+  scale_fill_arcadia(palette_name = "secondary", reverse = TRUE) +
+  scale_y_continuous(expand=c(0,0)) + # removes whitespace between axis and bars
+  theme(legend.position = "bottom")
+```
+In addition to reversing the order of the colors used in the palette, you can select different indices of colors from the palettes within the `scale` function with the `start` and `end` arguments:
+```{r, indices}
+ggplot(mtcars, aes(x = hp, fill = as.factor(cyl))) +
+  geom_density(alpha = 0.8, linewidth = 0) + # remove border line from filled-in density plots
+  theme_arcadia() +
+  scale_fill_arcadia(palette_name = "blue_shades", start=2, end=5) +
+  scale_y_continuous(expand=c(0,0)) +
+  scale_x_continuous(expand=c(0,0)) # remove whitespace between both axes and the plot
+```
+The `scale` functions are used to specify which palettes to use. The `gradient` functions are used to access the gradient palettes, which work in the same way as the `scale` functions using `color` or `fill`:
+```{r, gradient}
+ggplot(data = mtcars, aes(x = hp, y = mpg, color = hp)) +
+ geom_point(size=2.5) + 
+ theme_arcadia() + 
+ gradient_color_arcadia(palette_name = "lisafrank")
+```
+There are also bicolor gradients available that are useful for heatmap plots. You can also remove the background color with `background=FALSE`, which is recommended when exporting plots, which is described below. 
```

Comment:
Given that this paragraph references a bicolor gradient, should "blue_red" or something else be used instead of "reds" below?

Reply:
addressed with `purplegreen`, it looks a little weird on my screen because I think these values are both in light areas

Reply:
Hmm... I think the gradient itself is broken somehow; taking a look at the color scale, it's got a super odd-looking banding pattern.

![Screenshot 2024-07-29 at 3 16 40 PM](https://github.com/user-attachments/assets/73a9d931-230e-4ecc-9773-b5819e7cc555)

Might this be an issue with the underlying package?

Reply:
The single color ones look find but I think there's something wrong with the bicolor ones. I'll change this here back to the single color and drop the mention of bicolor and open an issue to figure out what happened with the positions here